TeslaSUI/TeslaSUI/Views/ClimateView.swift

```diff
@@ -0,0 +1,393 @@
+// ClimateView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран настроек климат контроля
+struct ClimateView: View {
+    // MARK: - Public Propeties
+    
+    var body: some View {
+        backgroundStackView {
+            ZStack {
+                VStack {
+                    HStack {
+                        backButtonView
+                        Spacer()
+                        screenNameTextView
+                        Spacer()
+                        settingsButtonView
+                    }
```

Comment:
```suggestion
                    HStack {
                        backButtonView
                        Spacer()
                        screenNameTextView
                        Spacer()
                        settingsButtonView
                    }
```
вынести в проперти, а в идеале в отдельный файлик, так как точно такая же вью будет на следующем экране